build.gradle

```diff
@@ -227,6 +234,28 @@ subprojects {
                 compileClasspath += generated.output
                 runtimeClasspath += generated.output
             }
+            scala {
+                compileClasspath += generated.output
```

Comment:
Why was this necessary? Doesn't it aggregate them in?

Reply:
It totally does. Not sure why this was added. Let me try to remove it to see what happens. It everything works, I'll take it out, otherwise I'll report back with what broke.

Reply:
Seems to work.